sympa.spec

```diff
@@ -76,6 +75,7 @@ BuildRequires: perl(Cwd)
 BuildRequires: perl(Data::Password)
 BuildRequires: perl(DateTime)
 BuildRequires: perl(DateTime::Format::Mail)
+BuildRequires: perl(DBD::SQLite)
```

Comment:
I guess this is a new requirement from the test suite ?

Reply:
Yes, it is used by a new test `t/02_Sympa_DatabaseManager.t` in sympa-community/sympa#336